models/db/world.ts

```diff
@@ -1,11 +1,12 @@
 import { Types } from 'mongoose';
 import User from './user';
-
+import Level from './level'
 // represents a document from the pathology.worlds collection
 export default interface World {
   _id: Types.ObjectId;
   authorNote?: string;
   name: string;
   psychopathId?: number;
   userId: Types.ObjectId & User;
+  levels: Types.Array<Level>;
```

Comment:
nit: alphabetical
there will also be lint warnings in the build for sorting the imports

Reply:
Fixed